frontend/app/package.json

```diff
@@ -62,6 +62,7 @@
     "cronstrue": "^2.47.0",
     "dagre": "^0.8.5",
     "jotai": "^2.6.0",
+    "js-confetti": "^0.12.0",
```

Comment:
the most important thing 😁

Reply:
https://www.loom.com/share/9309721b45ff43079f2b8125696f6a9a?sid=1ba0264a-77ec-4b06-a800-e7e235bbdca0

dangerous confetti ;)